### PR TITLE
content: publish jtag article #5

### PR DESCRIPTION
**Pull request by bahildebrand**

### CODE REVIEW

**Comment by bahildebrand** [comment]
@Zamuhrishka would you be able to look over some of the grammar/spelling changes I made. Also worth noting that the youtube links are still broken. I'm looking into how to get those properly embedded, but I'd appreciate your eyes on the rest of the content.

File: _drafts/diving-into-jtag-part5.md [hunk 1]
```diff
@@ -70,147 +67,147 @@ a TAP, it will be put into the `BYPASS` state.
 
 We specify [STM32F405_415_407_417_LQFP100.bsd](https://bsdl.info/details.htm?sid=61a8799988cb03f688ca59b002289d77)
 file for the first TAP - SGS/Thomson(`06413041h`) as it is responsible for Boundary Scan. We leave the second TAP
-in `BYPASS`. To select a file, click on the *CLICK HERE TO SET* link and select the desired file.
+in `BYPASS .`To select a file, click the *CLICK HERE TO SET* link and select the desired file.
```
Comment (gminn) [inline _drafts/diving-into-jtag-part5.md:70]: Looks like we actually want to make this:
```suggestion
in `BYPASS`. To select a file, click the *CLICK HERE TO SET* link and select the desired file.
```

File: _drafts/diving-into-jtag-part5.md [hunk 2]
```diff
@@ -8,12 +7,11 @@ tags: [arm, cortex-m, mcu, debugging, debugger]
...
-The main examples of JTAG usage, such as debugging and testing of boards in production,
-we have considered in the previous parts. And for firmware/embedded developers only the
-first example - debugging - is the most useful. As far as I know, we almost never encounter
-the second option, although it can be very useful too. And in this article I want to look
-at two examples of JTAG Boundary Scan which I think can be very useful in everyday work of
-a firmware/embedded developer: Bring-up and Revers Enginnering.
+In previous articles we have considered the main examples of JTAG usage, such as debugging
+and testing boards in production. For firmware/embedded developers, the
+first example - debugging - is the most useful. In this article, I want to look
+at two examples of JTAG Boundary Scan, which can be very useful in the everyday work of
```
Comment (gminn) [inline _drafts/diving-into-jtag-part5.md:13]: I like the rework but I think we could clarify the language some more. How about:
```suggestion
In previous articles, we have considered the primary uses of JTAG, including debugging
and testing boards in production. For firmware/embedded developers, the
first - debugging - is the most common. In this article, I want to look
at two uses of JTAG Boundary Scan, which are also common tasks for
a firmware/embedded developer: board bring-up and reverse engineering.
```

**Comment by Zamuhrishka** [comment]
> @Zamuhrishka would you be able to look over some of the grammar/spelling changes I made. Also worth noting that the youtube links are still broken. I'm looking into how to get those properly embedded, but I'd appreciate your eyes on the rest of the content.

Everything looks good for me. Thank you for your work.

**Comment by Zamuhrishka** [comment]
> Got a few fixes, thank you for reviewing this 🙏

Thank you, Gillian! Sorry for my English. :)

**Review by gminn — APPROVED** [review]

**Pull request merged**